docs/SQL/gems/datasources/upload-files.md

```diff
@@ -30,6 +30,7 @@ The recommended maximum file size is 100&nbsp;MB.
 
 :::
 
+- Open a Model within the Project
```

Comment:
I suggest we merge this line into the leading line above it.

So, "To upload your file, start by opening a Model within a project and following one of these steps:"